server/app/services/geo/AddressCandidates.java

```diff
@@ -0,0 +1,34 @@
+package services.geo;
+
+import com.google.auto.value.AutoValue;
+import com.google.common.collect.ImmutableList;
+
+@AutoValue
+public abstract class AddressCandidates {
+  public static Builder builder() {
+    return new AutoValue_AddressCandidates.Builder();
+  }
+
+  /**
+   * this method returns a well-known ID for ArcGIS coordinate systems, used for spatial reference
```

Comment:
suggestion (blocking): remove "this class" and "this method" portions of javadoc in this file as per [style guide](https://google.github.io/styleguide/javaguide.html#s7.2-summary-fragment)